docs/solutions/workflow/gt-modify-no-c-flag-silent-unstaged-miss.md

```
@@ -0,0 +1,86 @@
+track: bug
+problem: gt modify without -c amends prior commit and ignores unstaged file edits; output looks successful
+tags: [graphite, gt-modify, unstaged, commit, silent-failure]
+components: [workflow, graphite]
+---
+
+## Problem
+
+Running `gt modify -m "fix: ..."` after applying a file edit (e.g., via the Edit
+tool) produces output like:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)